src/libraries/System.Collections/tests/Generic/List/List.Generic.Tests.Find.cs

```diff
@@ -981,6 +998,23 @@ public void FindAll_VerifyDuplicates(int count)
             VerifyList(list.FindAll(_alwaysFalseDelegate), new List<T>());
         }
 
+        [Fact]
+        public void FindAll_ListSizeCanBeChanged()
+        {
+            var list = new List<int>() { 1, 2, 3 };
+            var results = list.FindAll(i =>
+            {
+                if (i < 4)
+                {
+                    list.Add(i + 1);
+                }
+
+                return true;
+            });
+
+            Assert.Equal(list, results);
```

Comment:
This looks like an exact duplicate of the test added earlier?

Reply:
Yeah, you are right. My bad.